index.html

```diff
@@ -0,0 +1,217 @@
+                  alt=""
+                  class="program-icon"
+                />
+                <h3>Mental health</h3>
+              </div>
+              <div class="program-details">
+                The program aim to promote psychological well-being, alleviate
+                distress, and improve overall quality of life.
+              </div>
+              <div class="card-desription"></div>
+            </div>
+            <div class="program-card">
+              <div class="icon">
+                <img
+                  src="./images/icons/medical.png"
+                  alt=""
+                  class="program-icon"
+                />
+                <h3>Mental health</h3>
+              </div>
+              <div class="program-details">
+                The program aim to promote psychological well-being, alleviate
+                distress, and improve overall quality of life.
+              </div>
+              <div class="card-desription"></div>
+            </div>
+            <div class="program-card">
+              <div class="icon">
+                <img
+                  src="./images/icons/blood-test.png"
+                  alt=""
+                  class="program-icon"
+                />
+                <h3>Mental health</h3>
+              </div>
+              <div class="program-details">
+                The program aim to promote psychological well-being, alleviate
+                distress, and improve overall quality of life.
+              </div>
+              <div class="card-desription"></div>
+            </div>
+          </div>
+          <u><p>See the whole program</p></u>
+          <button
+            type="button"
+            class="appointment-btn active-btn-effect:active hover-btn-effect"
+          >
+            Make Appointment
+          </button>
+        </section>
+        <section class="team">
+          <div class="section-title">
+            <h2 id="team-section-header">Our Team</h2>
+            <hr />
+          </div>
+        </section>
+      </main>
+    </div>
+    <footer class="section-partners">
+      <div class="section-title">
+        <h2 class="section-header">Partner</h2>
+        <hr />
+      </div>
+
+      <div class="partners-wrapper">
+        <ul class="partner">
+          <li>
+            <img src="./images/logo/google.png" alt="" class="partner-logo" />
+          </li>
+          <li>
+            <img
+              src="./images/logo/coca-cola.png"
+              alt=""
+              class="partner-logo"
+            />
+          </li>
+          <li>
+            <img src="./images/logo/espn.png" alt="" class="partner-logo" />
+          </li>
+          <li>
+            <img src="./images/logo/ford.png" alt="" class="partner-logo" />
+          </li>
+          <li>
+            <img src="./images/logo/rainbow.png" alt="" class="partner-logo" />
+          </li>
+        </ul>
+      </div>
+    </footer>

```

Comment:
- [ ] Kindly modify the footer tag to a normal section (the partner section) and create a proper footer tag with the logo to the left side and some text aligned to the right as described in the design about page. Please note that both pages should have the same navbar and footer. 🙏 
![Document](https://github.com/FilbertKimasa/Mobile-clinic/assets/47625407/077ba870-bbf5-478e-b7b0-0c942a59ad85)